docs/tutorials/setup-env.md

```diff
@@ -0,0 +1,165 @@
+---
+id: setup-env
+title: Environment Setup 
+---
+This tutorial will help you set up your local development environment to build your first DHIS2 app using the [DHIS2 Application Platform](https://platform.dhis2.nu/#/), which provides a fully-featured DHIS2 app, with out-of-the-box tools and components ready to be customized for your use case. 
+
+In this section we will:
+
+1. Install the [DHIS2 CLI](https://cli.dhis2.nu/#/)
+2. Spin up a DHIS2 instance with [d2 cluster](https://cli.dhis2.nu/#/commands/d2-cluster) 
```

Comment:
Since many people might use an existing DHIS2 instance, rather than spinning up their own, maybe we should separate out the Cluster tutorial (which we kind of already have [here](https://developers.dhis2.org/docs/guides/spin-up-local-instance) and keep the "setup environment" tutorial focused just on setting up the javascript environment (node, yarn, cli-style, etc) and connecting to another instance, then link to the cluster setup guide?

Reply:
I agree! I separated the cluster section :)